ui/app/AppLayouts/Profile/Sections/AdvancedContainer.qml

```diff
@@ -11,234 +11,80 @@ Item {
-        }
 
-        SVGImage {
-            id: caret3
-            anchors.right: parent.right
-            anchors.rightMargin: 0
-            anchors.verticalCenter: networkText.verticalCenter
-            source: "../../../img/caret.svg"
-            width: 13
-            height: 7
-            rotation: -90
-        }
-        
-        ColorOverlay {
-            anchors.fill: caret3
-            source: caret2
-            color: Style.current.darkGrey
-            rotation: -90
-        }
-
-        NetworksModal {
-            id: networksModal
+        StatusSettingsLineButton {
+            text: qsTr("Communities")
+            isSwitch: true
+            switchChecked: appSettings.communitiesEnabled
+            onClicked: function (checked) {
+                appSettings.communitiesEnabled = checked
+            }
         }
 
-        MouseArea {
-            anchors.fill: parent
-            onClicked: networksModal.open()
-            cursorShape: Qt.PointingHandCursor
+        StatusSettingsLineButton {
+            text: qsTr("Node Management")
+            isSwitch: true
+            switchChecked: appSettings.nodeManagementEnabled
+            onClicked: function (checked) {
+                appSettings.nodeManagementEnabled = checked
+            }
         }
     }
 
-    Item {
-        id: fleetSetting
-        anchors.top: networkTabSettings.bottom
-        anchors.topMargin: 20
-        anchors.left: parent.left
-        anchors.leftMargin: 24
-        width: parent.width - 20
-        height: fleetText.height
-
-        StyledText {
-            id: fleetText
-            text: qsTr("Fleet")
-            font.pixelSize: 15
-        }
-
-        StyledText {
-            text: profileModel.fleets.fleet
-            font.pixelSize: 15
-            anchors.right: caret2.left
-            anchors.rightMargin: Style.current.padding
-        }
-
-        SVGImage {
-            id: caret2
-            anchors.right: parent.right
-            anchors.rightMargin: 0
-            anchors.verticalCenter: fleetText.verticalCenter
-            source: "../../../img/caret.svg"
-            width: 13
-            height: 7
-            rotation: -90
-        }
-        
-        ColorOverlay {
-            anchors.fill: caret2
-            source: caret2
-            color: Style.current.darkGrey
-            rotation: -90
-        }
-
-        FleetsModal {
-            id: fleetModal
-        }
-
-        MouseArea {
-            anchors.fill: parent
-            onClicked: fleetModal.open()
-            cursorShape: Qt.PointingHandCursor
-        }
+    NetworksModal {
+        id: networksModal
     }
 
-    RowLayout {
-        id: uiCatalog
-        anchors.top: fleetSetting.bottom
-        anchors.topMargin: 20
-        anchors.left: parent.left
-        anchors.leftMargin: 24
-        Component.onCompleted: {
-            uiComponentBtn.enabled = false
-        }
+    FleetsModal {
+        id: fleetModal
+    }
```

Comment:
Maybe we should put those Modals in `Component`s, that way they are not initiated at app launch, but instead only when the button is clicked to open them? You can use openPopup` for that https://github.com/status-im/nim-status-client/blob/master/ui/app/AppMain.qml#L31

Don't forget in that case to add an `onDestroy` to the popups